installer/cli/scenario2cli.py

```diff
@@ -0,0 +1,73 @@
+import click
+from installer.cli.click_required import Required
+from installer.configurators.jenkins_container import configure_jenkins_container
+from installer.configurators.bitbucket import configure_bitbucket
+from installer.configurators.sonarqube_container import configure_sonarqube_container
+from installer.helpers.terraform import exec_terraform_apply
+
+
+@click.command()
+@click.option(
+    '--bitbucket_endpoint',
+    required=True,
+    help='Provide the endpoint (without the protocol or port) of the Bitbucket server to be configured',
+    prompt=True
+)
+@click.option(
+    '--bitbucket_userpass',
+    nargs=2,
+    required=True,
+    help='Provide the username and password of the existing Bitbucket administrator separated by a space',
+    prompt=True
+)
+@click.option(
+    '--bitbucket_publicip',
+    required=True,
+    help='Provide the public IP of the Bitbucket server to be configured.',
```

Comment:
"Provide the DNS name ..." so that we are consistent

Reply:
Fixed.